CRM/Core/SelectValues.php

```diff
@@ -550,7 +550,7 @@ public static function eventTokens() {
       '{event.contact_phone}' => ts('Event Contact Phone'),
       '{event.description}' => ts('Event Description'),
       '{event.location}' => ts('Event Location'),
-      '{event.fee_amount}' => ts('Event Fees'),
```

Comment:
This differed from the tokens - I picked the other in this case - it will probably change again before we are done as this should be on the participant tokens class